storybook/stories/core/components/Input/PasswordInput.stories.js

```diff
@@ -17,6 +17,12 @@ export default {
         type: {summary: 'Set custom style to the input'},
       },
     },
+    strength: {
+      control: {type: 'number'},
```

Comment:
Better set a select control since number can be arbitrary. 
https://github.com/orangehrm/oxd/blob/ent/storybook/stories/core/components/Alert/Alert.stories.js#L23

Reply:
strength: {
      control: {type: 'select', options: [0, 1, 2, 3, 4, 5]},
      table: {
        type: {summary: 'Set the strength of the password'},
      },